pkg/stub/stub.go

```diff
@@ -504,7 +505,7 @@ func (stub *stub) Run(ctx context.Context) error {
 
 	err = <-stub.srvErrC
 	if err == ttrpc.ErrServerClosed {
-		return nil
```

Comment:
if we swallow the error, plugin authors will not be able to implement the logic to handle it and it will be assumed everything exited clean